INSTALL.md

```diff
@@ -26,6 +26,14 @@ osm2pgsql --output flex --style openstreetmap-carto.lua -d flex ~/path/to/data.o
 
 You can find a more detailed guide to setting up a database and loading data with osm2pgsql at [switch2osm.org](https://switch2osm.org/serving-tiles/manually-building-a-tile-server-16-04-2-lts/).
 
+### Disable JIT
+
+We do not recommend [PostgreSQL JIT](https://www.postgresql.org/docs/current/jit-reason.html), which is on by default in PostgreSQL 12 and higher. JIT is benifitial for slow queries where executing the SQL takes substantial time and that time is not spent in function calls. This is not the case for rendering, where most time is spent either fetching from disk, in PostGIS functions, or the query is fast. In theory, the query planner will only use JIT on slower queries, but it is known to get the type of queries map rendering requries wrong.
```

Comment:
Typo: 'benifitial' 
Typo: 'requries'

Reply:
I haven't changed this file - this is from the merge from master. Can you open an issue to fix this in master?